Casks/m/meta-quest-developer-hub.rb

```diff
@@ -0,0 +1,24 @@
+cask "meta-quest-developer-hub" do
+  version "4.8.0"
+  sha256 :no_check
+
+  url "https://securecdn.oculus.com/binaries/download/?id=26027232766892142"
+  name "meta-quest-developer-hub"
+  desc "VR development tool"
+  homepage "https://developer.oculus.com/meta-quest-developer-hub/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
+  end
+
+  depends_on macos: ">= :catalina"
+
+  app "Meta Quest Developer Hub.app"
+
+  # No zap stanza required
```

Comment:
```suggestion
  zap trash: [
    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/com.oculus.odh.sfl*",
    "~/Library/Application Support/Meta Quest Developer Hub",
    "~/Library/Application Support/odh",
    "~/Library/Preferences/com.oculus.odh.plist",
    "~/Library/Saved Application State/com.oculus.odh.savedState",
  ]
```

The comment `# No zap stanza required` should only be used if no additional files can be found.

Reply:
Weird, using [@nrlquaker's homebrew/createzap](https://github.com/nrlquaker/homebrew-createzap) I got this output: 

```
No meta-quest-developer-hub settings found.
Instead of a zap stanza, include the following comment:
# No zap stanza required
```

Reply:
@thatrobotdev Sorry, I didn't notice your comments initially, as the issue was already labeled as resolved.

Regarding your question: the tool can sometimes be inaccurate, so you may need to perform a manual search (see [documentation](https://docs.brew.sh/Cask-Cookbook#zap-creation)).

Reply:
No worries, thanks for the info!